.github/workflows/ci.yml

```diff
@@ -48,6 +48,7 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python-version }}
+          allow-prereleases: true
```

Comment:
Python 3.13 has been released, so we don't need to enable pre-releases anymore.

Reply:
This will be useful for 3.14 anyway, no? Not a huge deal, we can add it back later.